Zpp/Simulation/Agents/JobDistributor/JobDistributorSkills.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections.Generic;
+using Akka.Actor;
+using AkkaSim.Definitions;
+using Master40.DB.DataModel;
+using Zpp.Common.ProviderDomain.Wrappers;
+using Zpp.Mrp.MachineManagement;
+using Zpp.Simulation.Agents.JobDistributor.Types;
+
+namespace Zpp.Simulation.Agents.JobDistributor
+{
+    partial class JobDistributor
+    {
+        public enum Command
+        {
+            GetWork
+        }
+        public class OperationsToDistibute : SimulationMessage
```

Comment:
1 file, one class, I know it's not enforced by C#, but recommended though. Is is really much more work to create a file per class?

Reply:
This will may change in the future. Currently its just a collection of wrappers for simulation messages that belong to each typespecific agent.

Reply:
Ok.